src/Microsoft.DocAsCode.Plugins/Manifest.cs

```diff
@@ -18,6 +18,10 @@ public class Manifest
         [JsonProperty("homepages")]
         public List<HomepageInfo> Homepages { get; set; }
 
+        [YamlMember(Alias = "sourceBasePath")]
+        [JsonProperty("source_base_path")]
```

Comment:
>source_base_path [](start = 23, length = 16)

have a question, why not use "sourceBasePath" to be consitent with yaml name and below xrefmap?